docs/modules/ROOT/pages/api/udc.adoc

```diff
@@ -0,0 +1,86 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= Universal Deployer
+
+Reference of the Universal Deployer Contract (UDC) interface and preset contract.
+
+== Core
+
+[.contract]
+[[IUniversalDeployer]]
+=== `++IUniversalDeployer++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.10.0/src/utils/universal_deployer/interface.cairo#L7[{github-icon},role=heading-link]
+
+```javascript
+use openzeppelin::utils::universal_deployer::interface::IUniversalDeployer;
```

Comment:
As suggested in another comment, I think we should change the structure of the utils to match:

`use openzeppelin::utils::interfaces::IUniversalDeployer;`

Reply:
Done!